dipy/reconst/tests/test_shore_metrics.py

```diff
@@ -74,8 +74,7 @@ def test_shore_metrics():
 
     # compare the shore pdf with the ground truth multi_tensor pdf
 
-    sphere = get_sphere('symmetric724')
-    v = sphere.vertices
+    v = default_sphere.vertices
```

Comment:
I wonder if the change here caused the CI error on Python 3.5. I just restarted that build, to seehow intermittent that is.